cdc/log.hh

```diff
@@ -89,9 +92,10 @@ public:
         lowres_clock::time_point timeout,
         utils::chunked_vector<mutation>&& mutations,
         tracing::trace_state_ptr tr_state,
-        db::consistency_level write_cl
-        );
-    bool needs_cdc_augmentation(const utils::chunked_vector<mutation>&) const;
+        db::consistency_level write_cl,
+        std::optional<per_request_options> options = {}
+    );
+    bool needs_cdc_augmentation(const utils::chunked_vector<mutation>&, const std::optional<per_request_options>&) const;
```

Comment:
Nit: not sure we should add the parameter to `needs_cdc_augmentation` in this PR, it is not used yet.